tgui/packages/tgui/interfaces/OreBox.tsx

```diff
@@ -43,6 +44,16 @@ export const OreBox = (props, context) => {
                 </Table.Cell>
               </Table.Row>
             ))}
+            {boulders > 0 ? (
```

Comment:
```suggestion
            {boulders > 0 && (
```
Won't need to : null this way